docs/content/cheatsheet.md

```diff
@@ -0,0 +1,1159 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: '0.8'
+    jupytext_version: 1.4.1+dev
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(myst_cheatsheet)=
+# MyST Cheat Sheet
+
+## Headers
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 15 15
+
+* - Text
+  - Example
+  - Note
+* - ```md
+    # Heading level 1
+    ## Heading level 2
+    ### Heading level 3
+    #### Heading level 4
+    ##### Heading level 5
+    ###### Heading level 6
+    ```
+  - ```md
+  	# MyST Cheat Sheet
+    ```
+  - Level 1-6 headings, denoted by number of `#`
+``````
+
+## Target Headers
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 15 15
+
+* - Text
+  - Example
+  - Note
+* - ```md
+    (target_header)=
+    ```
+  - ```md
+    (myst_cheatsheet)=
+    # MyST Cheat Sheet
+    ```
+  - See {ref}`below <ref/target_headers>` how to reference target headers.
+``````
+
+(ref/target_headers)=
+### Referencing Target Headers
+
+Targets can be referenced with the [ref inline role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref) which by default uses the section title:
+```md
+{ref}`myst_cheatsheet`
+```
+You can specify the text of the target:
+```md
+{ref}`MyST syntax lecture <myst_cheatsheet>`
+```
+Another alternative is to use markdown syntax:
+```md
+[MyST syntax lecture](myst_cheatsheet)
+```
+
+## Quote
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 20 10
+
+* - Text
+  - Example
+  - Note
+* - ```md
+    > text
+    ```
+  - ```md
+    > this is a quote
+    ```
+  - quoted text
+``````
+
+## Thematic Break
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 15 15
+
+* - Text
+  - Example
+  - Note
+* - ```md
+    ---
+    ```
+  - ```md
+    This is the end of some text.
+
+    ---
+
+    ## New Header
+    ```
+  - Creates a horizontal line in the output
+``````
+
+## Line Comment
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 15 15
+
+* - Text
+  - Example
+  - Note
+* - ```md
+    % text
+    ```
+  - ```md
+    a line
+    % a comment
+    another line
+    ```
+  - See [Comments](https://myst-parser.readthedocs.io/en/latest/using/syntax.html#syntax-comments) for more information.
+``````
+
+## Block Break
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 15 15
+
+* - Token
+  - Example
+  - Result
+* - ```md
+    +++
+    ```
+  - ```md
+    This is an example of
+    +++ {"meta": "data"}
+    a block break
+    ```
+  - This is an example of
+    +++ {"meta": "data"}
+    a block break
+``````
+
+## HTML Block
+
+``````{list-table}
+:header-rows: 1
+:widths: 20 15 15
+
+* - Token
```

Comment:
What's the deciding factor between when to call it "text" and when to call it "token"?

Reply:
Ah nice catch. I think maybe we should change "text" and "token" to "syntax"?

Reply:
that works for me - most importantly we should be consistent

Reply:
resolved here 5c204b1